lib/consumer.js

```diff
@@ -55,7 +53,7 @@ Consumer.prototype.getRegistrar = function() {
 Consumer.prototype.getRegistrant = function(address) {
   var self = this;
   return self.registrar.registrantIndexAsync(address, {from: self.address}).then(function(index) { 
```

Comment:
These async functions, are they defined in web3 from contract methods now or does Promisify provide them out of the box?